website/docs/docs/collaborate/git/resolve-merge-conflicts.md

```diff
@@ -1,70 +1,66 @@
 ---
-title: "Resolve merge conflicts"
+title: "Resolve merge conflicts in the Cloud IDE"
 id: "resolve-merge-conflicts"
 ---
 
-Merge conflicts often occur when multiple users are concurrently making edits to the same section in the same file. This makes it difficult for Git to determine which change should be kept. The merge conflict process gives users the ability to sort out which lines of code should be kept and committed. Here we'll show you how you'd resolve merge conflicts in the IDE!
+[Merge conflicts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts) often occur when multiple users are concurrently making edits to the same section in the same file. 
 
-## Running into a merge conflict
+This makes it difficult for Git to decide what changes to incorporate in the final merge.  The Cloud IDE merge conflict process gives you the ability to decide which lines of code to keep and commit. This document will show you how you can resolve merge conflicts in the Cloud IDE.
 
-In this example we have a column that represents the total number of orders your customer has ever had. It's currently named `number_of_orders` but your stakeholder feels like this could be named better.
+## Identify merge conflicts
 
-You and a teammate make changes to the column name, but have gone two separate routes. You rename this column to `total_number_of_orders` and your teammate has renamed it `historical_order_count`.
+You can experience a merge conflict in two possible ways:
 
-Your teammate has committed their change first, so you encounter a merge conflict when you press commit.
+- Pulling changes from your main branch when someone else has merged a conflicting change.
+- Committing your changes to the same branch when someone else has already committed their change first.
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-ide/commit-to-merge-conflict.png" title="Commit which will trigger the merge conflict"/>
+The way to [resolve](#resolving-conflicts) either scenario will be exactly the same. 
+
+For example, if you and a teammate make changes to the same file and commit, you will encounter a merge conflict as soon as you **Commit and push**. The Cloud IDE will display:
+
+
+- The git action bar **Commit and resolve** under **Version Control**  instead of `Commit`. This indicates that the Cloud IDE has detected some conflicts that you need to address.
```

Comment:
Slight adjustment:

The git action bar will display `Commit and resolve` under **Version Control**  instead of `Commit`.